### PR TITLE
Adaptions for 10.9

### PR DESCRIPTION
**Pull request by BobSilent**
This PR contains the required changes for updating to jellyfin 10.9.

Currently no nuget packages are available - or at least i did not found any for 10.9 therefore this will not build.
But who wants to build can disabled these nuget references
```
    <PackageReference Include="Jellyfin.Data" Version="10.*-*" />
    <PackageReference Include="Jellyfin.Controller" Version="10.*-*" />
    <PackageReference Include="Jellyfin.Common" Version="10.*-*" />
    <PackageReference Include="Jellyfin.Model" Version="10.*-*" />
```

and add directly references to these dlls from a 10.9 installation:
- Jellyfin.Data.dll
- Jellyfin.Extensions.dll
- MediaBrowser.Common.dll
- MediaBrowser.Controller.dll
- MediaBrowser.Model.dll

### CODE REVIEW

**Comment by scampower3** [comment]
Hey, `serviceCollection.AddHostedService<TvdbMissingEpisodeProvider>();` is needed to be added in RegisterServices in TvdbPluginServiceRegistrator.cs

File: Jellyfin.Plugin.Tvdb/Providers/TvdbMissingEpisodeProvider.cs [hunk 1]
```diff
@@ -128,7 +118,7 @@ private bool IsEnabledForLibrary(BaseItem item)
             }
 
             var libraryOptions = _libraryManager.GetLibraryOptions(series);
-            return _baseItemManager.IsMetadataFetcherEnabled(series, libraryOptions, ProviderName);
+            return _baseItemManager.IsMetadataFetcherEnabled(series, libraryOptions.GetTypeOptions(item.GetType().Name), ProviderName);
```
Comment (scampower3) [inline Jellyfin.Plugin.Tvdb/Providers/TvdbMissingEpisodeProvider.cs:121]:
```suggestion
            return _baseItemManager.IsMetadataFetcherEnabled(series, libraryOptions.GetTypeOptions(series.GetType().Name), ProviderName);
```

Comment (scampower3) [reply Jellyfin.Plugin.Tvdb/Providers/TvdbMissingEpisodeProvider.cs:121]: If not IsMetadataFetcherEnabled will always be false.

**Comment by crobibero** [comment]
@BobSilent when you have a moment can you please fix merge conflicts?

We are trying something new :eyes:

**Comment by BobSilent** [comment]
@crobibero: forgot to mention - done!

**Pull request closed without merging**